tests/phpunit/Fixtures/Mutation/FakeNodeTraverser.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Infection\Tests\Fixtures\Mutation;
+
+use LogicException;
+use PhpParser\Node;
+use PhpParser\NodeTraverserInterface;
+use PhpParser\NodeVisitor;
+
+final class FakeNodeTraverser implements NodeTraverserInterface
```

Comment:
those `Fake*` classes allow to simplify some tests by not requiring to declare a mock with no expected calls. So it saves a bit of boiler template (which makes tests harder to read) and also speeds up things a bit (mocks are slower)